src/background/handlers/getMostVisitedSites.js

```diff
@@ -0,0 +1,16 @@
+import { db } from '../initdb.js';
+// this can be optimized i just dont know how
```

Comment:
you can probably use `each` instead of `toArray` to stream through items instead of loading them all